tools/pd-ctl/README.md

```diff
@@ -266,12 +266,6 @@ The configuration above is global. You can also tune the configuration by config
     config set high-space-ratio 0.5             // Set the threshold value of sufficient space to 0.5
     ```
 
-- `disable-raft-learner` is used to disable Raft learner. By default, PD uses Raft learner when adding replicas to reduce the risk of unavailability due to downtime or network failure.
```

Comment:
Is it better to add comment to describe that the option has been deprecated in Readme?

Reply:
Do you mean leave it as before and just mark this option as deprecated?

Reply:
yep